package.json

```diff
@@ -15,6 +15,7 @@
 		"@typescript-eslint/eslint-plugin": "^2.21.0",
 		"@typescript-eslint/parser": "^2.21.0",
 		"babel-eslint": "^10.1.0",
+		"checksum": "^0.1.1",
```

Comment:
Will get 3PP approval for this dependency.

Reply:
@gjung-sfdc Can we use the typed checksum dependency here so that we can follow typescript's way of importing dependencies?
https://www.npmjs.com/package/@types/checksum

Reply:
^ Roopa and I decided to stick with the js version for now.

Reply:
Can we use the built in `crypto` methods? https://odino.org/generating-the-md5-hash-of-a-string-in-nodejs/ has an example of generating an MD5.

```Javascript
const crypto = require('crypto')
let hash = crypto.createHash('md5').update('some_string').digest("hex")
```